From: Samuel Martin <s.martin49@gmail.com>
Date: Thu, 29 May 2014 19:22:27 +0200
Subject: [PATCH] auto/lib/libxslt/conf: use pkg-config

Change to using pkg-config to find the path to libxslt and its
dependencies.

Signed-off-by: Martin Bark <martin@barkynet.com>
---
 auto/lib/libxslt/conf | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/auto/lib/libxslt/conf b/auto/lib/libxslt/conf
index 3063ac7ce0c0..3209e3642fdc 100644
--- a/auto/lib/libxslt/conf
+++ b/auto/lib/libxslt/conf
@@ -12,8 +12,9 @@
                       #include <libxslt/xsltInternals.h>
                       #include <libxslt/transform.h>
                       #include <libxslt/xsltutils.h>"
-    ngx_feature_path="/usr/include/libxml2"
-    ngx_feature_libs="-lxml2 -lxslt"
+    ngx_feature_path="$(${PKG_CONFIG:=pkg-config} --cflags-only-I libxslt|
+			 sed -re 's/(^|\s)-I\s*(\S+)/\1\2/g')"
+    ngx_feature_libs="$(${PKG_CONFIG:=pkg-config} --libs libxslt)"
     ngx_feature_test="xmlParserCtxtPtr    ctxt = NULL;
                       xsltStylesheetPtr   sheet = NULL;
                       xmlDocPtr           doc = NULL;
